Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -21,6 +23,8 @@
 static const int CACHE_VERSION = 13;  // Last changed in PR #3261
 static const int DATASTREAM_VERSION = QDataStream::Qt_5_5;
 
+u64 GetTitleID(QString);
```

Comment:
This can just be defined here and marked as an internally linked function. The parameter can also be a const reference.